services/debugger/source/eventUploader.go

```diff
@@ -1,5 +1,6 @@
 package sourcedebugger
 
+//go:generate mockgen -destination=./mocks/mock.go -package=mocks github.com/rudderlabs/rudder-server/services/debugger/source SourceDebugger
```

Comment:
Do we need to generate a mock? The interface signature seems simple enough to mock by hand.

Reply:
Its simpler to just generate a mock, mock by hand feels too much manual work(in writing a mock as well as to maintain it)